src/main/java/de/retest/recheck/review/workers/SaveFilterWorker.java

```diff
@@ -1,42 +1,51 @@
 package de.retest.recheck.review.workers;
 
-import static de.retest.recheck.configuration.ProjectConfiguration.RECHECK_IGNORE;
-
+import java.io.File;
+import java.io.FileOutputStream;
 import java.io.IOException;
 import java.io.PrintStream;
-import java.nio.file.Files;
 import java.nio.file.Path;
 import java.util.Optional;
 import java.util.stream.Stream;
 
 import de.retest.recheck.ignore.CacheFilter;
 import de.retest.recheck.ignore.Filter;
 import de.retest.recheck.ignore.JSFilterImpl;
-import de.retest.recheck.ignore.RecheckIgnoreUtil;
+import de.retest.recheck.ignore.RecheckIgnoreLocator;
 import de.retest.recheck.review.GlobalIgnoreApplier;
 import de.retest.recheck.review.GlobalIgnoreApplier.PersistableGlobalIgnoreApplier;
 import de.retest.recheck.review.ignore.io.Loaders;
 
 public class SaveFilterWorker {
 
 	private final GlobalIgnoreApplier applier;
+	private final RecheckIgnoreLocator locator;
 
 	public SaveFilterWorker( final GlobalIgnoreApplier applier ) {
+		this( applier, new RecheckIgnoreLocator() );
+	}
+
+	protected SaveFilterWorker( final GlobalIgnoreApplier applier, final RecheckIgnoreLocator locator ) {
 		this.applier = applier;
+		this.locator = locator;
 	}
 
 	public void save() throws IOException {
-		final Optional<Path> ignoreFile = RecheckIgnoreUtil.getProjectIgnoreFile( RECHECK_IGNORE );
+		final Optional<Path> ignorePath = locator.getProjectIgnoreFile();
 		final PersistableGlobalIgnoreApplier persist = applier.persist();
 
+		final File ignoreFile = ignorePath.orElse( locator.getUserIgnoreFile() ).toFile();
+		if ( !ignoreFile.exists() ) {
+			ignoreFile.getParentFile().mkdirs();
+		}
```

Comment:
`FileUtils.openOutputStream` does this for you (if we do not want to use files).

Reply:
Which one? `org.apache.commons.io.FileUtils` is not...

Reply:
`org.apache.commons.io.FileUtils#openOutputStream´ when I look in my IDE

Reply:
Done